src/debugger.fs

```diff
@@ -44,28 +47,31 @@ module Debugger =
 module Program =
     open Elmish
 
+    let inline private getTransformersWith encoder decoder =
+        let deflate x =
+            try encoder x
+            with er ->
+                Debugger.showWarning [er.Message]
+                box x
+        let inflate x =
+            match Decode.fromValue "$" decoder x with
+            | Ok x -> x
+            | Error er -> failwith er
+        deflate, inflate
+
     let inline private getTransformers<'model>() =
-        let showEncodeError =
-            let mutable hasShownError = false
-            fun (er: System.Exception) ->
-                if not hasShownError then
-                    hasShownError <- true
-                    JS.console.warn("[ELMISH DEBUGGER]", er.Message)
         try
-            let encoder = Encode.Auto.generateEncoder<'model>()
-            let decoder = Decode.Auto.generateDecoder<'model>()
-            let deflate x =
-                try encoder x
-                with er ->
-                    showEncodeError er
-                    box x
-            let inflate x =
-                match Decode.fromValue "$" decoder x with
-                | Ok x -> x
-                | Error er -> failwith er
-            deflate, inflate
+            let coders =
+                Extra.empty
+                |> Extra.withDecimal
+                |> Extra.withInt64
+                |> Extra.withInt64
+                // |> Extra.withBigInt
```

Comment:
True, it's in separate lib, I'm not sure dragging bigint is worth it by default

Reply:
Well, decimals and longs are also in a separate lib now, but they're used much more frequently so...